internal/connection_repository/sql_connection_locator.go

```diff
@@ -134,21 +134,22 @@ func (scm *SqlConnectionLocator) GetConnectionsByAccount(ctx context.Context, ac
 
 	for rows.Next() {
 		var client_id domain.ClientID
+		var org_id domain.OrgID
```

Comment:
We've got to be careful here.  Until we perform the database migration where we map account numbers to org_ids, its possible for the org_id to be null.  This scan will return an error when it hits a record with a null org_id because it cannot convert a null to a string.  

Don't ask me how I know this.  ;)

I think the cleanest thing to do here is to read the org_id from the database as a ```sql.NullString```.  If its a valid string, then we can convert it to an domain.OrgID.

Let me know if that is not clear and we can discuss it.